test/dlc_tests/conftest.py

```diff
@@ -959,7 +1011,7 @@ def skip_pt21_test(request):
     if request.node.get_closest_marker("skip_pt21_test"):
         if Version(image_framework_version) in SpecifierSet("==2.1"):
             pytest.skip(
-                f"PT2.1 SM DLC doesn't support Rubik and Herring for now, so skipping this container with tag {image_framework_version}"
+                f"PT2.1 SM DLC doesn't support SMDMP and SMDDP for now, so skipping this container with tag {image_framework_version}"
```

Comment:
nit: This is an unclear skip message based on the fixture name, as the fixture implies that it can be used for any pt21 skip condition, but the message is coupled to DGL, SMDDP and SMMP.